spec/tapioca/gem/pipeline_spec.rb

```diff
@@ -3866,5 +3867,135 @@ module ViewHelpers::UrlHelper; end
 
       assert_equal(output, compile)
     end
+
+    it "compile RBIs with location from gem source" do
+      add_ruby_file("foo.rb", <<~RB)
```

Comment:
Can we break some of these constants into multiple "files" so that we can ensure that our file detection is working properly?